core/templates/dev/head/pages/exploration-editor-page/settings-tab/settings-tab.directive.ts

```diff
@@ -87,33 +87,33 @@ angular.module('oppia').directive('settingsTab', [
       controllerAs: '$ctrl',
       controller: [
         '$http', '$rootScope', '$scope', '$uibModal', '$window',
-        'AlertsService', 'ChangeListService',
-        'EditableExplorationBackendApiService',
+        'AlertsService',
+        'ChangeListService', 'EditableExplorationBackendApiService',
```

Comment:
The `'ChangeListService'` can fit on the previous line.

Reply:
Fixed.